<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateCompteDialog</class>
 <widget class="QDialog" name="updateCompteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mise a jour de compte</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Black</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(61, 48, 162);</string>
   </property>
   <property name="text">
    <string>Mise Ã  jour du Compte</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>261</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="updatePasswordBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color:white;
border:none;
border-radius:8px;
font-size:15px;
background-color: rgb(61, 48, 162);
font-weight:bold;
}</string>
      </property>
      <property name="text">
       <string>Enregistrer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="update_pass_concel_btn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color:white;
border:none;
border-radius:8px;
font-size:15px;
background-color: rgb(187, 187, 187);
font-weight:bold;
}</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>87</y>
     <width>501</width>
     <height>223</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Ancien mot de passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="update_old_pass_lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat SemiBold</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Nouveau mot de passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="update_new_pass_lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat SemiBold</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Confirmer mot de passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="update_confirm_pass_lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat SemiBold</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
